jkube-kit/config/image/src/test/java/org/eclipse/jkube/kit/config/image/build/HealthCheckConfigTest.java

```diff
@@ -92,65 +93,79 @@ public void testGoodHealthCheck7() {
                 .validate();
     }
 
-    @Test(expected = IllegalArgumentException.class)
+    @Test
     public void testBadHealthCheck1() {
         HealthCheckConfiguration.builder()
+        Assertions.assertThrows(IllegalArgumentException.class, () -> {
                 .mode(HealthCheckMode.none)
                 .interval("2s")
                 .build()
                 .validate();
+         });
```

Comment:
You've broken the code, you might want to change your lambda statement to something like this.

```suggestion
        final HealthCheckConfiguration hcc = HealthCheckConfiguration.builder()
            .mode(HealthCheckMode.none)
            .interval("2s")
            .build()
        Assertions.assertThrows(IllegalArgumentException.class, () -> hcc.validate());
```